nestkernel/synaptic_element.cpp

```diff
@@ -47,7 +47,7 @@ nest::SynapticElement::SynapticElement()
   , z_connected_( 0 )
   , continuous_( true )
   , growth_rate_( 1.0 )
-  , tau_vacant_( 10.0 )
+  , tau_vacant_( 0.1 )
```

Comment:
Hi dear @heplesser, yes you are right, the change is intentional. This value was unused, but it was part of the original design. This change was done in order to adapt the value to the current implementation of the decay.